sdk/synapse/synapse-artifacts/review/synapse-artifacts.api.md

```diff
@@ -168,9 +168,7 @@ export interface AmazonRdsForSqlServerSource extends TabularSource {
     produceAdditionalTypes?: any;
     sqlReaderQuery?: any;
     sqlReaderStoredProcedureName?: any;
-    storedProcedureParameters?: {
-        [propertyName: string]: StoredProcedureParameter;
-    };
+    storedProcedureParameters?: any;
```

Comment:
Could you share the swagger link related to this change with me?

Reply:
swagger definition:
https://github.com/Azure/azure-rest-api-specs/tree/main/specification/synapse/data-plane

Reply:
@MaryGao The change is made in this PR
https://github.com/Azure/azure-rest-api-specs/commit/9bcba68f1b66a11d4e29dc333e973ac494a1c9a9#diff-c3ceb0562d85d027038cc0405e265fecd17cc9de1d0cc0da54f072509d825d65